backend/cellguide/pipeline/computational_marker_genes/computational_markers.py

```diff
@@ -139,27 +62,28 @@ def __init__(self, *, snapshot: WmgSnapshot, all_cell_type_ids_in_corpus: list[s
             for k, v in i.items()
         }
         self.gene_id_to_symbol.update(primary_filters__gene_id_to_symbol)
+
         # Groupby variables used to group the data various operations
         # cell types are removed as they are treated differently
         # all other metadata (like tissue and organism) are dimensions across
         # which marker gene computation for cell types will be stratified
         if "cell_type_ontology_term_id" in groupby_terms:
             groupby_terms.remove("cell_type_ontology_term_id")
-        else:
-            raise ValueError("cell_type_ontology_term_id must be one of the groupby terms")
 
         self.groupby_terms = groupby_terms
         self.groupby_terms_with_celltype = groupby_terms + ["cell_type_ontology_term_id"]
         self.groupby_terms_with_celltype_and_gene = self.groupby_terms_with_celltype + ["gene_ontology_term_id"]
 
         # load the cell counts and expression summary cubes fully in memory
         cell_counts_df = snapshot.cell_counts_cube.df[:]
-        expressions_df = snapshot.expression_summary_fmg_cube.df[:]
+        expressions_df = snapshot.expression_summary_default_cube.df[:]
 
         # prep the cell counts and expressions dataframes
-        (self.cell_counts_df, self.expressions_df) = self._prepare_cell_counts_and_gene_expression_dfs(
-            cell_counts_df, expressions_df
-        )
+        (
+            self.cell_counts_df,
+            self.cell_counts_df_orig,
```

Comment:
cell_counts_df_orig is used to subtract out counts of overlapping descendants from comparisons